AzureCalling/AppDelegate.swift

```diff
@@ -22,17 +22,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
-
```

Comment:
Could you confirm with Alex that if our sample app only supports portrait mode?

Reply:
> Could you confirm with Alex that if our sample app only supports portrait mode?

Good point. I'll check with Alex and per discussion, we won't block this PR to wait for a reply. We'll create a seperate PR for supporting landscape if needed.